hummingbot/strategy/arbitrage/arbitrage.pyx

```diff
@@ -95,7 +95,6 @@ cdef class ArbitrageStrategy(StrategyBase):
         self._last_timestamp = 0
         self._next_trade_delay = next_trade_delay_interval
         self._last_trade_timestamps = {}
-        self._exchange_rate_conversion = ExchangeRateConversion.get_instance()
```

Comment:
why was this removed/refactored?

Reply:
It is a singleton class and we don't need an instance stored in the strategy instance. One of the strategy method is moved outside of the strategy class to be re-used in discovery strategy.